applications/FluidDynamicsApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -180,6 +181,17 @@ void  AddCustomUtilitiesToPython(pybind11::module& m)
         .def_static("CalculateFluidNegativeVolume", &FluidAuxiliaryUtilities::CalculateFluidNegativeVolume)
         ;
 
+    py::class_<FluidTestUtilities>(m, "FluidTestUtilities")
+        .def_static("RandomFillHistoricalVariable", &FluidTestUtilities::RandomFillHistoricalVariable<double>)
+        .def_static("RandomFillHistoricalVariable", &FluidTestUtilities::RandomFillHistoricalVariable<array_1d<double, 3>>)
+        .def_static("RandomFillNonHistoricalVariable", [](ModelPart::NodesContainerType& rNodesContainer, const Variable<double>& rVariable, const IndexType DomainSize, const double MinValue, const double MaxValue) { FluidTestUtilities::RandomFillNonHistoricalVariable(rNodesContainer, rVariable, DomainSize, MinValue, MaxValue);})
+        .def_static("RandomFillNonHistoricalVariable", [](ModelPart::NodesContainerType& rNodesContainer, const Variable<array_1d<double, 3>>& rVariable, const IndexType DomainSize, const double MinValue, const double MaxValue) { FluidTestUtilities::RandomFillNonHistoricalVariable(rNodesContainer, rVariable, DomainSize, MinValue, MaxValue);})
+        .def_static("RandomFillNonHistoricalVariable", [](ModelPart::ConditionsContainerType& rConditionsContainer, const Variable<double>& rVariable, const IndexType DomainSize, const double MinValue, const double MaxValue) { FluidTestUtilities::RandomFillNonHistoricalVariable(rConditionsContainer, rVariable, DomainSize, MinValue, MaxValue);})
+        .def_static("RandomFillNonHistoricalVariable", [](ModelPart::ConditionsContainerType& rConditionsContainer, const Variable<array_1d<double, 3>>& rVariable, const IndexType DomainSize, const double MinValue, const double MaxValue) { FluidTestUtilities::RandomFillNonHistoricalVariable(rConditionsContainer, rVariable, DomainSize, MinValue, MaxValue);})
+        .def_static("RandomFillNonHistoricalVariable", [](ModelPart::ElementsContainerType& rElementsContainer, const Variable<double>& rVariable, const IndexType DomainSize, const double MinValue, const double MaxValue) { FluidTestUtilities::RandomFillNonHistoricalVariable(rElementsContainer, rVariable, DomainSize, MinValue, MaxValue);})
+        .def_static("RandomFillNonHistoricalVariable", [](ModelPart::ElementsContainerType& rElementsContainer, const Variable<array_1d<double, 3>>& rVariable, const IndexType DomainSize, const double MinValue, const double MaxValue) { FluidTestUtilities::RandomFillNonHistoricalVariable(rElementsContainer, rVariable, DomainSize, MinValue, MaxValue);})
+        ;
+
```

Comment:
I'm strongly against this random-based testing. I'd use deterministic values, specially for the elemental testing to ease an eventual debugging. You can achieve the same by creating an auxiliary function that implements the test and then calling it with a small set of deterministic values.